src/components/utils/form/MultiOptionAutocomplete.js

```diff
@@ -20,6 +20,7 @@ const MultiOptionAutocomplete = forwardRef(
         className,
         disabled,
         inputProps,
+        margin,
```

Comment:
I think a more flexible solution is to replace this with a `textFieldProps`, and in the last argument of `TextField`, use `...textFieldProps`